### PR TITLE
implemented custom GTM

### PR DESCRIPTION
**Pull request by svenlatham**
Added custom GTM so we can dynamically control cookies and links

### CODE REVIEW

**Comment by svenlatham** [comment]
Notes for this implementation.

The Javascript provided is given to us by Google. [Some detail here](https://london.atlassian.net/wiki/spaces/DAT/pages/3893264406/Cookies)

Google Tag Manager is a scripted tool that allows for custom HTML and other tools to be injected into the page. It is particularly useful for addition of other scripts, so in our case can be used as an entry point for Google Analytics and Civic Cookie Control.

By way of demonstrating its functionality, while this PR is active, you should see the following text inserted into the very bottom of the page: This text confirms that Google Tag Manager has been correctly installed. This test will be removed following validation.

**Review by madelinekosse — APPROVED** [review]
I'm not sure what the reasoning is behind this, but this seems to add some javascript that does indeed run on the page so approved, assuming this is indeed the javascript you want to add. Obviously need to remove the footer text before merging

**Comment by svenlatham** [comment]
Enables us to test Civic Cookie code in a controlled way on the live domain.

**Pull request merged**
